### PR TITLE
add support for different os systems

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
add support for different os systems

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by JasonWu0506** [comment]
it looks different from the original files? we defined 3 levels of quality, but I didn't see the definition in the file diff.

File: br/br-deployment.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ summary: 了解如何部署和使用 BR。
 
 ## 部署 BR
 
-- 生产环境中，推荐 BR 运行在（8 核+/16 GB+）的节点上。 操作系统版本要求可参考 [Linux 操作系统版本要求](/hardware-and-software-requirements.md#linux-操作系统版本要求)。
+- 生产环境中，推荐 BR 运行在（8 核+/16 GB+）的节点上。操作系统版本要求可参考 [TiDB 对操作系统及平台的要求](/hardware-and-software-requirements.md#对操作系统及平台的支持)。
```
Comment (TomShawn) [inline br/br-deployment.md:12]:
```suggestion
- 生产环境中，推荐 BR 运行在（8 核+/16 GB+）的节点上。操作系统版本要求可参考 [TiDB 对操作系统及平台的要求](/hardware-and-software-requirements.md#对操作系统及平台的要求)。
```

File: hardware-and-software-requirements.md [hunk 2]
```diff
@@ -6,7 +6,49 @@ title: TiDB 软件和硬件环境建议配置
 
 TiDB 作为一款开源分布式 NewSQL 数据库，可以很好地部署和运行在 Intel 架构服务器环境、ARM 架构的服务器环境及主流虚拟化环境，并支持绝大多数的主流硬件网络。作为一款高性能数据库系统，TiDB 支持主流的 Linux 操作系统环境。
 
-## Linux 操作系统版本要求
+## 对操作系统及平台的要求
```
Comment (lilin90) [inline hardware-and-software-requirements.md:9]: 以与其它小节的用词风格保持一致
```suggestion
## 操作系统及平台要求
```

File: br/br-deployment.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ summary: 了解如何部署和使用 BR。
 
 ## 部署 BR
 
-- 生产环境中，推荐 BR 运行在（8 核+/16 GB+）的节点上。 操作系统版本要求可参考 [Linux 操作系统版本要求](/hardware-and-software-requirements.md#linux-操作系统版本要求)。
+- 生产环境中，推荐 BR 运行在（8 核+/16 GB+）的节点上。操作系统版本要求可参考 [TiDB 对操作系统及平台的要求](/hardware-and-software-requirements.md#对操作系统及平台的要求)。
```
Comment (TomShawn) [inline br/br-deployment.md:12]:
```suggestion
- 生产环境中，推荐 BR 运行在（8 核+/16 GB+）的节点上。操作系统版本要求可参考 [TiDB 对操作系统及平台的要求](/hardware-and-software-requirements.md#操作系统及平台要求)。
```

File: hardware-and-software-requirements.md [hunk 4]
```diff
@@ -6,7 +6,49 @@ title: TiDB 软件和硬件环境建议配置
...
+
+    > **注意：**
+    >
+    > 根据 [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/)，CentOS 的上游支持已于 2021 年 12 月 31 日终止。
+
++ 在以下操作系统以及对应的 CPU 架构组合上，你可以编译构建和部署 TiDB，可使用 OLTP 和 OLAP 以及数据工具的基本功能。但是 TiDB **不保障企业级生产质量要求**：
+
+    |  操作系统   |   支持的 CPU 架构   |
+    |   :---   |   :---   |
+    |   MacOS Catalina 及以上的版本  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
```
Comment (TomShawn) [inline hardware-and-software-requirements.md:33]:
```suggestion
    |   macOS Catalina 及以上的版本  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
```

File: hardware-and-software-requirements.md [hunk 5]
```diff
@@ -6,7 +6,49 @@ title: TiDB 软件和硬件环境建议配置
...
+    | <ul><li>Red Hat Enterprise Linux 8.4 及以上的 8.x 版本</li><li>CentOS 8.4 及以上的 8.x 版本</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
+    | <ul><li>Red Hat Enterprise Linux 7.3 及以上的 7.x 版本</li><li>CentOS 7.3 及以上的 7.x 版本</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>   |
+    | 麒麟欧拉版 V10 SP1/SP2   |   <ul><li>x86_64</li><li>ARM 64</li></ul>   |
+    | UOS V20                 |   <ul><li>x86_64</li><li>ARM 64</li></ul>   |
+
+    > **注意：**
+    >
+    > 根据 [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/)，CentOS 的上游支持已于 2021 年 12 月 31 日终止。
+
++ 在以下操作系统以及对应的 CPU 架构组合上，你可以编译构建和部署 TiDB，可使用 OLTP 和 OLAP 以及数据工具的基本功能。但是 TiDB **不保障企业级生产质量要求**：
```
Comment (lilin90) [inline hardware-and-software-requirements.md:29]:
```suggestion
+ 在以下操作系统以及对应的 CPU 架构组合上，你可以编译、构建和部署 TiDB，可使用 OLTP 和 OLAP 以及数据工具的基本功能。但是 TiDB **不保障企业级生产质量要求**：
```

File: hardware-and-software-requirements.md [hunk 6]
```diff
@@ -6,7 +6,49 @@ title: TiDB 软件和硬件环境建议配置
...
 
-## Linux 操作系统版本要求
+## 操作系统及平台要求
+
+<SimpleTab>
+<div label="v6.1.1 及以上的 v6.1.x 版本">
+
+自 v6.1.1 开始，针对不同操作系统和 CPU 架构的组合，TiDB 提供不同级别质量标准的支持。
+
++ 在以下操作系统以及对应的 CPU 架构组合上，TiDB 可满足符合企业级生产质量的要求，产品特性经过全面且系统化的验证：
```
Comment (lilin90) [inline hardware-and-software-requirements.md:16]:
```suggestion
+ 在以下操作系统以及对应的 CPU 架构组合上，TiDB 可**满足企业级生产质量的要求**，产品特性经过全面且系统化的验证：
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Comment by JasonWu0506** [comment]
LGTM

**Pull request merged**